### ISSUE TITLE
"You're hosting 65.5MB of files", while no files are actually hosted

### ISSUE THREAD

**Issue by RubenKelevra** [ipfs/ipfs-webui#1757]
**Describe the bug**
Currently, the UI shows several MB of data being "hosted" while there are no pins, no MFS-files and the GC has been run.

The "data" is probably just DHT stuff, being stored - but it's confusing for the users.

Not sure if this is a go-ipfs limitation or a WebUI one, but it should show 0 Bytes of there's no actual data shared with the network.

**Desktop (please complete the following information):**
 - OS: ArchLinux
 - Browser: Chrome, Firefox
 - Version: latest

**Comment by michel47** [comment]
Well after an init, the ipfs node stores https://gateway.ipfs.io/ipfs/QmQPeNsJPyVWPFDVHb77w8G42Fvo15z4bG2X8D2GhfbSXc
which is 6.5KB and the empty root directory ([BCIQF...X3Y](https://gateway.ipfs.io/ipfs/BCIQFTFEEHEDF6KLBT32BFAGLXEZL4UWFNWM4LFTLMXQBCERZ6CMLX3Y)): 4B so that should be about the minimum you could have stored ...

![screenshot1.png](https://gateway.ipfs.io/ipfs/QmS8ZGNG7jNVp6fyXrTHcUhgxS7EoB2hjp8D3RForUiLPL/screenshot1.png)
![screenshot2.png](https://gateway.ipfs.io/ipfs/QmS8ZGNG7jNVp6fyXrTHcUhgxS7EoB2hjp8D3RForUiLPL/screenshot2.png)

however when it runs IPFS creates a minimum of 16.4KB
```
ipfs repo stat
NumObjects: 1
RepoSize:   16854
StorageMax: 10000000000
RepoPath:   .
Version:    fs-repo@11
```

**Comment by lidel** [comment]
This should be fixed in  [v2.12.0](https://github.com/ipfs/ipfs-webui/releases/tag/v2.12.0), which  simplified the way header of _Files_ looks like:

Hovering on "Files" shows explainer:

> ![2021-04-19--22-47-49](https://user-images.githubusercontent.com/157609/115301345-597a0280-a161-11eb-91dd-3de7aba57038.png)

Hovering on "All blocks" shows:

>  ![2021-04-19--22-47-31](https://user-images.githubusercontent.com/157609/115301366-5f6fe380-a161-11eb-9484-d2b9259f92f4.png)
